.github/workflows/vuln-scan.yaml

```diff
@@ -171,7 +174,7 @@ jobs:
       # ref: https://github.com/jacobtomlinson/gha-find-replace
       - name: Update VULN_SCAN_TIME in Dockerfile
         if: steps.analyze.outputs.proceed == 'yes'
-        uses: jacobtomlinson/gha-find-replace@f9e200cf233bcde71011fa5e4178037881764379 # associated tag: 0.1.3
+        uses: jacobtomlinson/gha-find-replace@f9e200cf233bcde71011fa5e4178037881764379
```

Comment:
Is this action just doing an in-place sed?

Reply:
Yepp, it can certainly be replaced a `sed` script

Reply:
Do you agree that we should stop using a dedicated action for this and use a `sed` script or similar instead, that doesn't involve a dedicated action that could contain a vulnerability etc?